src/scripts.js

```diff
@@ -60,8 +66,51 @@ function updateAllRecipeDisplay() {
 
     // TO DO: Put modal event handler in line 63
     recipeCard.onclick = () => {
+      // if (event classname === star-icon) {
+        // do the function to toggleFavorites
+      // }
+
+      // if (event classname === recipe-tags) {
+    //  the function tagsToggleFilter
+      // }
+
+       // if (event classname === recipe-card) {
+    //  the function to open the recipe modal
+      // }
+
       console.log(`hi! ${recipe.name}`)
     }
     recipeSection.appendChild(recipeCard)
   })
 }
+
+function tagsToggleFilter(event) {
+  if(tag === event.target.innerText) {
+    updateAllRecipeDisplay(recipeRepo.allRecipes)
+    event.target.style.backgroundColor = "cornflowerblue";
```

Comment:
We need to add here setting `tag` back to undefined or an empty string, so that a user can click on a tag a third time and refilter all the recipes